lib/charms/tls_certificates_interface/v2/tls_certificates.py

```diff
@@ -308,7 +308,7 @@ def _on_all_certificates_invalidated(self, event: AllCertificatesInvalidatedEven
 
 # Increment this PATCH version before using `charmcraft publish-lib` or reset
 # to 0 if you are raising the major API version
-LIBPATCH = 14
+LIBPATCH = 15
```

Comment:
We have merged this [bug fix](https://github.com/canonical/tls-certificates-interface/pull/75
We had to merge this bug fix).
Please rebase and bump the LIBPATCH accordingly.

Reply:
Done.